hack/README.md

```diff
@@ -0,0 +1,64 @@
+# Useful resources for development on `lakefs-spec`
+
+This document contains information on the resources in this directory, and how they can be used in development and testing.
+
+## `docker-compose.yml` - local lakeFS quickstart instance
+
+This Docker Compose file bootstraps a local [lakeFS quickstart instance](https://docs.lakefs.io/quickstart/launch.html).
+It does not come with an associated volume, so you can spin it up and down without creating dangling resources.
+
+Requirements:
+* A Docker runtime & CLI with `docker compose`.
+
+To bootstrap, run the following command:
+
+```shell
+docker compose -f hack/docker-compose.yml up
+```
+
+To stop the container again, run
+
+```shell
+docker compose -f hack/docker-compose.yml down
+```
+
+## `lakefs-s3-local.yml` - lakeFS with a local, SeaweedFS-backed S3 blockstore
+
+For simulating a lakeFS deployment with a remote blockstore, the `lakefs-s3-local.yml` Docker Compose file contains a
+recipe mocking an S3 blockstore using SeaweedFS.
```

Comment:
Add a link to the project web site for reference, probably not a lot of people have heard of it before.

Also, I wouldn't call it "mocking", since it is a proper S3 implementation.

Reply:
I linked to the wiki in [852ff8c](https://github.com/appliedAI-Initiative/lakefs-spec/pull/115/commits/852ff8c6426c61bca7f4dabb31cc43cad4e3ac15), since the landing page immediately links there and does not contain anything else.